### DISCUSSION TITLE
API to view logs

### DISCUSSION THREAD

**Discussion by gaganbiswas** [pocketbase/pocketbase#6037] [Q&A] [answered]
Is there any list/view API or any way through which I can get the logs and also add API rules as to who can view the logs?

**Comment by ganigeorgiev** [comment, marked as answer]
Logs can be accessed only by superusers, aka. there are no API rules for them.

For the currently available logs Web APIs please check https://pocketbase.io/docs/api-logs/.

**Reply by gaganbiswas** [reply]
Is this same for previous version, i.e., v0.22.x.?

**Reply by ganigeorgiev** [reply]
Yes, https://pocketbase.io/old/docs/api-logs/.

**Reply by gaganbiswas** [reply]
Thank you ^^
